.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: dart
 sudo: false
 dart:
-  - stable
   - dev
-  - 1.23.0
+  - 2.0.0-dev.23.0
```

Comment:
Sorry, I didn't explain this very well but it should be the minimum supported version of the SDK. So in this case 2.0.0-dev.12.0.

Reply:
I did that at first, but that old version of the Dart SDK has a pub bug where it doesn't handle 2.0.0-dev SDK constraints correctly. I'm not sure exactly which version fixed that, so I bumped the minimum up to the version number that pub was getting stuck on.